README.md

```diff
@@ -75,6 +77,8 @@ amount of data to fine-tune on. For details see the
 
 
 [`timm`]: https://github.com/rwightman/pytorch-image-models
+[sayakpaul/collections/vision_transformer]: https://tfhub.dev/sayakpaul/collections/vision_transformer
```

Comment:
MLP-Mixers are also available: https://tfhub.dev/sayakpaul/collections/mlp-mixer/1. They were converted using the same procedure.

Reply:
Could you also add a table with results like for `vision_transformer`?

Reply:
For ImageNet-1k models, I can. I think I will just take the numbers from the paper or do you want me to run the evaluation on ImageNet-1k independently? 

This might take a day. So, would suggest keeping this PR open. Do you mind?

Reply:
@andsteing added some results. Here is the PR: https://github.com/tensorflow/tfhub.dev/pull/89.

Reply:
Thanks, added.

It would be great to have the table the same way as for the ViT models (i.e. also a i1k evaluation with TensorBoard link using the hub module)

Reply:
Yeah, I have added that. The format is a bit different because the corresponding conversion notebook enlists all the models very clearly and is easier to follow along (it's mentioned from the collection page which is to be updated by the PR).

Reply:
Ah ok, makes sense, thanks.